profile/src/main/java/rdx/works/profile/domain/backup/CloudBackupFileEntity.kt

```diff
@@ -0,0 +1,98 @@
+package rdx.works.profile.domain.backup
+
+import com.google.api.services.drive.model.File
+import com.radixdlt.sargon.Profile
+import com.radixdlt.sargon.ProfileId
+import com.radixdlt.sargon.Timestamp
+import kotlinx.serialization.Serializable
+import rdx.works.core.domain.cloudbackup.GoogleDriveFileId
+import rdx.works.core.sargon.ProfileIdSerializer
+import rdx.works.profile.domain.backup.CloudBackupFileEntity.Companion.LAST_MODIFIED_KEY
+import rdx.works.profile.domain.backup.CloudBackupFileEntity.Companion.LAST_USED_ON_DEVICE_KEY
+import rdx.works.profile.domain.backup.CloudBackupFileEntity.Companion.TOTAL_NUMBER_OF_ACCOUNTS_KEY
+import rdx.works.profile.domain.backup.CloudBackupFileEntity.Companion.TOTAL_NUMBER_OF_PERSONAS_KEY
+import java.time.format.DateTimeFormatter
+
+@Serializable
+data class CloudBackupFileEntity(
```

Comment:
shouldn't we explicitly name properties in serializable classes to avoid surprises with proguard?

Reply:
@Serilizable here is for nav args, do we really need the name property? 
I am fine to add it anyway.

>avoid surprises with proguard

we have proguard rules for kotlinx serialization

Reply:
ahh ok, then I guess it is fine